database.js

```diff
@@ -0,0 +1,67 @@
+var DBWrapper = require('node-dbi').DBWrapper;
+var config= require('./.databaseConfig');
+var moment = require('moment');
+var dbWrapper;
+
+
+module.exports = {
+	setup: (callback) => {
+		//setup the database connection config
+		var dbConnectionConfig = {host:config.db.host, user:config.db.username, password:config.db.password, database:config.db.database };
+		dbWrapper = new DBWrapper('pg', dbConnectionConfig);
+		//connect to the database
+		dbWrapper.connect();
+		callback(null, true);
+	},
+
+	getAllUsers: (callback) => {
+		dbWrapper.fetchAll('SELECT * FROM users', null, (err, res) => {
```

Comment:
Would you consider using an ORM instead of directly querying the database like this?